### ISSUE TITLE
How to rotate composite input just before composition?

### ISSUE THREAD

**Issue by Mustafax06** [lovell/sharp#1876]
I'm wondering is there a future to rotate a composite input image? I can rotate first and then I can use the rotated input but I'm looking for a simpler way to achieve this. 

    ` const back = path.resolve(__dirname, './raw/amplifiers/body.png');
      const front = path.resolve(__dirname, './raw/knobs/mouth.png'); // rotate this

      sharp(back)
        .composite([{ input: front, gravity: 'southeast' }]) // I want to rotate only this input...
        .webp({ quality: 100 })
        .toFile('output.png')`

**Comment by lovell** [comment]
Your second question at #1877 would suggest you found the answer - is this correct?

**Comment by Mustafax06** [comment]
> Your second question at #1877 would suggest you found the answer - is this correct?

Thank you for your answer and I'm really sorry for the delay. I achieved the rotation problem using another library but I don't want to use another library. I've removed the black background with your suggestion at [ #1877](https://github.com/lovell/sharp/issues/1877). But I couldn't rotate input file of composite function. There's no any option to rotation. Right now I'm writing rotated image to a file but you know that's not efficient. And if I rotate image at composition progress that would be wonderful.
